scripts/buffer.pl

```diff
@@ -0,0 +1,560 @@
+use strict;
+use warnings;
+
+use vars qw($VERSION %IRSSI);
```

Comment:
Thanks Nei. I prefer `our` to `use vars`. I probably copied and pasted those first lines from an old script.

I removed the line containing `use vars`.